dataAccessObjects.py

```diff
@@ -315,6 +315,11 @@ def getFileInfo(self,textId):
             #TODO raise exception
             return None
 
+    def getEvent(self,enevtId):
+        sql = 'SELECT COUNT(*) from text_data where text_invisible_title="{eventId}"}'.format(
+            eventId=eventId)
+        return self.db.query(sql)
```

Comment:
Please use `self.db.queryOneValue(sql)` and change the return value in caller function.
Refer to #228